_posts/2019-02-11-lecture-08.md

```diff
@@ -0,0 +1,174 @@
+
+Example - If we have just two random variables X and Y(both observed) where X is the cause of Y and there is no confounder.So, $$ P(Y|do(X)) = P(Y|X) $$ and we know this is identifiable because for changing $$ P(Y|X) $$, we would have to change the distribution of P(X,Y).
+
+But, now lets say we have a confounder C, which is the common cause of both X and Y. And this is not observed. Now, we can change the distribution of C such that P(X,Y) is unchanged but we get different $$ P(Y|do(X)) $$
+$$ P(Y|do(X)) $$ can also be written as $$ P(y | \hat{x}) $$
+#### Back Door Criterion 
+A set of variables Z satisfies the back-door criterion relative to an ordered pair of variables $$ (X_i, X_j) $$ if
+
+* No node in Z is a descendant of $$ X_i $$
+* Z blocks every path between $$ X_i $$ and $$ X_j $$ that contains an arrow into $X_i$ (Hence the name back-door)
+
+If the set of nodes Z follows these two conditions then $$ \[P (y | \hat{x}) = \sum_z P(y|x,z)P(z) \] $$
+Same as we saw earlier in the case of kidney stone treatment. And thus $P(y|\hat{x})$ is now identifiable because of this back-door adjustment.\\
+Proof : left
+#### Front Door Criterion
+A set of variables Z satisfies the front-door criterion relative to an ordered pair of variables $$ (X_i, X_j) $$ if
+* Z intercept all directed paths from X to Y (hence the name front-door)
+* Z There is no back-door path from X to Z 
+* All back-door paths from Z to Y are blocked by X
+If the set of nodes Z follows these two conditions then $$ \[P (y | \hat{x}) = \sum_z P(z|x)\sum_{x'}P(y|x',z)P(x') \] $$
+Proof: left
+#### Propensity Score
+
+The Average Causal Effect (ACE), defined as: 
+$$ \mathbb{E} \left[ Y \ | \ do(x) \right] - 
+\mathbb{E} \left[ Y \ | \ do(x') \right] $$
+
+
+Given that $$ P(Y \ | \ do(x)) = \sum_{c} P(Y \ | \ x,c)P(c) $$ might not share the same $$ P(c) $$, one solution for an estimator of the ACE can be through direct matching. If $$ C $$ is a high dimensional space one can use the propensity score to achieve unconfoundedness and estimate ACE.
+### Causal Discovery
+
+Causal discovery relies on two assumptions:
+The Causal Markov condition: that each variable is independent of its non-descendants given its parents.
+
+Faithfulness condition: all observed (conditional) independencies are captured in the causal graph. Roughly speaking, faithfulness allow you to go back from independence to causal graph.
+
+The above two conditions allows us to determine a list of candidate causal structures, with the same causal skeleton. Following this, we need to determine the causal direction. This step involves finding v-structure and doing orientation propagation. 
+
+The following example as given in lecture for reasoning about independence conditions:
+
+Suppose you have four variables $$ X_1, X_2, X_3, X_4 $$ such that:
+\[X_1 \perp X_2\]
+
+$$ X_1 \perp X_4 | X_3 $$
+
+$$ X_2 \perp X_4 | X_3 $$
```

Comment:
This equation doesn't render, please fix.

Changing "|" to "\mid" in the LaTeX code seems to fix the rendering problem elsewhere, so try that here.